modules/router/manifests/nginx.pp

```diff
@@ -76,9 +76,9 @@
   $check_requests_warning = '@25',
   $check_requests_critical = '@10',
   $robotstxt = '',
-  $proxy_busy_buffers_size = '8k',
-  $proxy_buffers = '8 4k',
-  $proxy_buffer_size = '4k',
+  $proxy_busy_buffers_size = '16k',
+  $proxy_buffers = '8 8k',
+  $proxy_buffer_size = '8k',
```

Comment:
These changes look good 👍 

It does look like the comments above the class need updating too

Reply:
🤦 done